tekore/_convert.py

```diff
@@ -159,6 +161,8 @@ def from_url(url: str) -> Tuple[str, str]:
     *prefixes, type_, id_ = url.split('/')
     prefix = '/'.join(prefixes)
 
+    id_ = id_.split("?")[0]
```

Comment:
Actually this could be more strict still. Something like `id_, _ = id_.split('?si=')` would allow for simultaneously checking that the parameter actually is the tracking parameter and there is only one `?si=`. I think that would be a nice addition, since we aren't expecting any other parameters. Also let's have single quotes for consistency. Sorry for the nitpick, this will improve once we start using Black 😅

Reply:
If we do `id_, _ = id_.split('?si=')` then we end up with `ValueError: not enough values to unpack (expected 2, got 1)` if there is no query parameters in the URL.

Reply:
I also see one potential problem by splitting on "?si=", and that is if they add yet another parameter, and we get a query string like `?pi=x&si=a`.  
Actually, I just looked up what a Spotify track URL ends up as if it's copied from Facebook, and it ends up with a `fbclid` parameter in it. By opening it I also get redirected to the album page, and the final URL ends up as `https://open.spotify.com/album/3Fi3WdHpIc7onkGv9s0u7I?highlight=spotify:track:6PmsvMNbxoQ1EOxSZu8fCY`

Reply:
Which brings up another question, how would you parse something like `https://open.spotify.com/album/3Fi3WdHpIc7onkGv9s0u7I?highlight=spotify:track:6PmsvMNbxoQ1EOxSZu8fCY`?
Should it return the album or track id? This is more difficult than I initially thought 😅

Reply:
Oh that's true, for some reason that totally slipped my mind. Damn. And yeah, knowing that some parameters get added I'm quite fine with stripping all the parameters!

Reply:
I think the album should be primary, spotify probably just uses that internally in their application. Definitely the album should be returned.